parts/masterparams.t

```diff
@@ -79,6 +79,13 @@
         "description": "A string hash of the master DNS name to uniquely identify the cluster."
       },
       "type": "string"
+    },
+		"{{.Name}}osDiskVhdUri": {
```

Comment:
@seanknox there is no type `.Name` in `MasterProfile` which is why you're getting an error. This can probably just be called `osDiskVhdUri`